scripts/mockapi.js

```diff
@@ -74,19 +74,6 @@ async function fetchRecommendations(type) {
   }));
 }
 
-async function fetchRapidResultMockData() {
```

Comment:
this can be part of pr https://github.com/aemsites/icicidirect/pull/226.

Reply:
yes, I am doing it. @JiangLong2019 you can remove this change from your PR as I am already removing conflicts.

Reply:
Okay, I'll roll back the changes around mock api.